qa-tests-backend/src/test/groovy/SACv2Test.groovy

```diff
@@ -1,126 +0,0 @@
-import static io.stackrox.proto.storage.RoleOuterClass.Access.NO_ACCESS
```

Comment:
Pretty sure this file is not deleted but rather empty. Since we don't need it anymore, should just delete it.

Reply:
Good catch, fixed!